README.md

```diff
@@ -1,3 +1,11 @@
 # Collaboration Example Repository
 
 This is a sample for the It Project in order to get familiar with GitHub.
+
+## Repository owern
```

Comment:
Does this really help? How about creating extra document for contributing guideline